pep-9999.rst

```diff
@@ -0,0 +1,214 @@
+PEP: 9999
+Title: Adding Vulnerability Data for the Simple API for Package Indexes
+Author: Dustin Ingram <di@python.org>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 28-Dec-2022
+
+
+Abstract
+========
+
+:pep:`691` defined a JSON form for the "Simple Repository API". This allowed
+clients to more easily query the data that was previously only available in
+HTML, as defined in :pep:`503`.
+
+This proposal adds new fields to the JSON form, which allow it to be used in
+place of PyPI's `JSON API <https://warehouse.pypa.io/api-reference/json.html>`__
+to acquire data about known vulnerabilities for a given project.
+
+The new fields are all part of the data returned from the "project details" URL.
+
+The schema for the vulnerability data and related fields is based on the `Open
+Source Vulnerability Schema <https://ossf.github.io/osv-schema/>`__, provided
+by the `Open Source Vulnerability (OSV) <https://osv.dev/>`__ project of the
+`Open Source Software Security Foundation (OpenSSF) <https://openssf.org/>`__.
+
+
+Rationale
+=========
+
+With the introduction of the JSON form of the simple API in :pep:`691`, the
+simple API offers functionality that is almost as complete as the PyPI JSON API.
+This PEP adds data which were previously only available through the JSON API,
+in order to allow more clients which were previously Warehouse specific to
+support arbitrary standards-compliant indexes.
+
+
+Specification
+=============
+
+This specification defines version 1.2 of the simple repository API. For the
+HTML version of the API, there is no change from version 1.0. For the JSON
+version of the API, the following changes are made:
```

Comment:
Just out of curiosity: why 1.2 instead of 1.1?

Reply:
1.1 is already specified: https://peps.python.org/pep-0700/

Reply:
Whoops, missed that completely 😅